design-system/pkg/src/password-field/docs/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: PasswordField
+description: Password fields are text fields for entering secure text.
+category: Forms
+---
+
+## Example
+
+```jsx {% live=true %}
+<PasswordField label="Password" />
+```
+
+## Related
+
+- [TextField](/package/text-field) — Use for typical text input.
+
+## Props
+
+### Label
+
+Every search field must have a label.
```

Comment:
I think this is meant to be `password field`?

```suggestion
Every password field must have a label.
```

Reply:
Good catch, thanks!